# See: https://github.com/bioconda/bioconda-recipes/pull/61815
# POSIX realtime extension library (librt) is not available on OSX, and removing it does not prevent
# building.
--- src/Makefile.rpsbproc.app
+++ src/Makefile.rpsbproc.app
@@ -30 +30 @@
-LIBS = $(ORIG_LIBS) $(CMPRS_LIBS) $(DL_LIBS) -lrt
+LIBS = $(ORIG_LIBS) $(CMPRS_LIBS) $(DL_LIBS)
